full-service/src/db/models.rs

```diff
@@ -68,24 +85,38 @@ pub struct NewAccount<'a> {
     pub name: &'a str,
 }
 
+/// A received transaction output entity that belongs to a an Account in this
+/// wallet. Also maybe a transaction output that hasn't been sent yet?
 #[derive(Clone, Serialize, Identifiable, Queryable, PartialEq, Debug)]
 #[primary_key(id)]
 pub struct Txo {
+    /// Primary key
     pub id: i32,
+    /// ???
     pub txo_id_hex: String,
+    /// The value of this transaction output, in picoMob.
     pub value: i64,
+    /// The serialized target_key of the TxOut.
     pub target_key: Vec<u8>,
+    /// The serialized public_key of the TxOut.
     pub public_key: Vec<u8>,
+    /// The serialized e_fog_hint of the TxOut.
     pub e_fog_hint: Vec<u8>,
+    /// The serialized TxOut.
     pub txo: Vec<u8>,
+    /// The receiving subaddress, if known.
     pub subaddress_index: Option<i64>,
+    /// TODO: why is this optional? Denoting if the Txo has subsequently been
```

Comment:
We actually precompute the KeyImage currently, so an unspent Txo could still have a `Some(key_image)`. I forget the reason for this...I recall I wanted to eliminate it, since at first I thought it was a security risk to have the precomputed key_images, but I think there's some reason it's necessary to store....which we should definitely document...

The reason it's optional is the same as the subaddress - you can only have the key_image if you also had the subaddress. So, if this field is None, then it's an "orphaned" txo.